content/actions/reference/workflow-syntax-for-github-actions.md

```diff
@@ -270,6 +270,20 @@ The jobs in this example run sequentially:
 2. `job2`
 3. `job3`
 
+#### Example not requiring dependent jobs to be successful
```

Comment:
I'll raise a docs issue for all the titles in this guide, so that we can move them to an "Example: `verb`" convention

Reply:
Done: https://github.com/github/docs/issues/2063 👍